lib/std/static_string_map.zig

```diff
@@ -329,33 +223,19 @@ fn testMap(map: anytype) !void {
     try testing.expect(null == map.get("averylongstringthathasnomatches"));
 }
 
-test "void value type, slice of structs" {
-    const slice = [_]TestKVVoid{
-        .{"these"},
-        .{"have"},
-        .{"nothing"},
-        .{"incommon"},
-        .{"samelen"},
-    };
-    const map = TestMapVoid.initComptime(slice);
-    try testSet(map);
-    // Default comparison is case sensitive
-    try testing.expect(null == map.get("NOTHING"));
-}
-
 test "void value type, list literal of list literals" {
     const slice = [_]TestKVVoid{
-        .{"these"},
-        .{"have"},
-        .{"nothing"},
-        .{"incommon"},
-        .{"samelen"},
+        .{ "these", {} },
+        .{ "have", {} },
+        .{ "nothing", {} },
+        .{ "incommon", {} },
+        .{ "samelen", {} },
```

Comment:
The previous API allowed to exclude void values from kvs_list with checks like this https://github.com/ziglang/zig/blob/master/lib/std/static_string_map.zig#L163.  I think restoring that behavior would be appreciated.